### ISSUE TITLE
[Help Wanted] Setup Error - File not found 'coreSchema.sql'

### ISSUE THREAD

**Issue by Mossaki** [OWASP/SecurityShepherd#773]
Hi there!
Trying to setup Security Shepherd locally on my PC. Running into an issue with the Database Setup Wizard for Security Shepherd. 
I keep getting this error:

Failed setting up the database: java.io.FileNotFoundException: File 'C:\Program%20Files\Apache%20Software%20Foundation\Tomcat%208.5\webapps\ROOT\WEB-INF\classes\database\coreSchema.sql' does not exist

But it does exist, I followed the file path and found what it was looking for but it isn't seeing it for some reason?

**PC Info**
- Windows 10
- Browser: Firefox

**Further Information:**
 - Tomcat 8.5
 - MariaDB 10.6
 - Shepherd Manual Pack
 - Java: JRE 1.8

**Screenshots**
![image](https://github.com/OWASP/SecurityShepherd/assets/149950111/1261177d-3beb-4b1a-b94b-8a96eccbc056)
![image](https://github.com/OWASP/SecurityShepherd/assets/149950111/c32303c9-0d4a-4520-a356-a5e8b4070349)

Anyone have an idea why its not working? Thanks!

**Comment by happyuser23** [comment]
Hi, it looks like the spaces in the directory name is represented by ```%20``` instead of the actual space character.
Have you tried to move / re-install Tomcat into another directory without spaces? I guess that may resolve your problem.

**Comment by Mossaki** [comment]
I'll try that, thank you!

**Comment by Mossaki** [comment]
> Hi, it looks like the spaces in the directory name is represented by `%20` instead of the actual space character. Have you tried to move / re-install Tomcat into another directory without spaces? I guess that may resolve your problem.

Omg this worked thank you so much!!

**Comment by emanet1** [comment]
Hi I want to find out from you if you have been using it without issue?

**Comment by Mossaki** [comment]
> Hi I want to find out from you if you have been using it without issue?

Yes, I got it to work! But I haven't been using it, I think I'm a bit out of my depth 😅

**Comment by SeanDuggan** [comment]
Closed - Thanks to happyuser23
